Stratis.Bitcoin.Features.Notifications/BlockNotification.cs

```diff
@@ -75,58 +72,70 @@ public virtual void SyncFrom(uint256 startHash)
 
         /// <summary>
         /// Notifies about blocks, starting from block with hash passed as parameter.
+        /// <para>
+        /// We don't need to wait for this asyncloop to complete as it doesn't rely or depend on
+        /// any other features finishing before shutting down. Stopping the node will stop the async 
+        /// loop task immediately.
+        /// </para>
         /// </summary>
-        public virtual IAsyncLoop Notify()
+        public void NotifyLoop()
         {
-            return this.asyncLoopFactory.Run("block notifier", token =>
+            this.asyncLoopFactory.Run("Notify", async token =>
```

Comment:
Don't you need the IAsyncLoop  to call dispose on shutdown?

Reply:
Dan, please see the comments on top of the method. We don't need to wait for the async loop to complete as with the ones as they don't depend on any other features completing first ;)

Reply:
@dangershony on looking at this again, I'm going to implement the dispose properly. We need to follow the pattern everywhere ;) Submitting soon.